app/views/pokemons/index.html.erb

```diff
@@ -1,17 +1,24 @@
+<h2>Pokemons in //location//</h2>
+<div class="container">
 
-<div class="search-results">
-  <%= @pokemons.each do |pokemon| %>
-  <div class="cards">
-    <div class="card-item">
-      <div>Photo</div>
-      <div class="description">
-        <%= pokemon.name %>
-        <%= pokemon.type %>
+  <div class="row">
+    <%= @pokemons.each do |pokemon| %>
+
+    <div class="col-12 col-lg-8">
```

Comment:
I think ` <div class="col-12 col-lg-8">` should possibly go outside of the .each loop and there's a lack of indentation at line 7 for what is being looped.